packages/components/src/components/Collapsible/elements.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import css from '@styled-system/css';
+
+export const Body = styled.div(
+  css({
+    borderBottom: '1px solid',
+    borderColor: 'sideBar.border',
+    paddingTop: 4,
+    paddingBottom: 8,
+  })
+);
+
+export const Header = styled.div(
+  css({
+    display: 'flex',
+    alignItems: 'center',
+    height: '32px',
+    paddingX: 3,
+    borderBottom: '1px solid',
```

Comment:
@SaraVieira This collapsible has too much information about the sidebar, should we just call it a SidebarSection instead, or should we break it into 2 parts where one non-styled part lives in `components` and the other extends in app/sidebar/?

Reply:
Yeah, when adding the animation I think it would make sense to call it `SidebarSection `